README.md

```diff
@@ -1,192 +1,74 @@
-
------- Training ------
-19/06/13 17:39:45 WARN Utils: Truncated the string representation of a plan since it was too large. This behavior can be adjusted by setting 'spark.debug.maxToStringFields' in SparkEnv.conf.
-Tracker started, with env={DMLC_NUM_SERVER=0, DMLC_TRACKER_URI=10.19.183.124, DMLC_TRACKER_PORT=9091, DMLC_NUM_WORKER=6}
-Elapsed time [train]: 152.021s
-
------- Transforming ------
-Elapsed time [transform]: 0.209s
-...
-...
-```
-
-You can try other apps by modifying the `--class` and `--trainDataPath`/ `--evalDataPath` parameters
-
-
-### supported parameters
-1. all [xgboost parameters](https://xgboost.readthedocs.io/en/latest/parameter.html) are supported
-2. -format=csv/parquet : The format of the data for training/transforming, now supports 'csv' and 'parquet' only. Required.
-3. -mode=all/train/transform. To control the behavior of the sample app, default is 'all' if not specified.
+1. All [xgboost parameters](https://xgboost.readthedocs.io/en/latest/parameter.html) are supported.
+2. `-format=csv/parquet`: The format of the data for training/transforming, now supports 'csv' and 'parquet' only. *Required*.
+3. `-mode=[all|train|transform]`. To control the behavior of the sample app, default is 'all' if not specified.
    * all: Do both training and transforming, will save model to 'modelPath' if specified
    * train: Do training only, will save model to 'modelPath' if specified.
    * transform: Do transforming only, 'modelPath' is required to locate the model data to be loaded.
-4. -trainDataPath=path : Path to your training data file(s), required when mode is NOT 'transform'.
-5. -trainEvalDataPath=path : Path to your data file(s) for training with evaluation. Optional.
-6. -evalDataPath=path : Path to your test(evaluation) data file(s), required when mode is NOT 'train'.
-7. -modelPath=path : Path to save model after training, or where to load model for transforming only. Required only when mode is 'transform'.
-8. -overwrite=true/false : Whether to overwrite the current model data under 'modelPath'. Default is false. You may need to set to true to avoid IOException when saving the model to a path already exists.
-9. -hasHeader=true/false : Indicate if your csv file has header.
+4. `-trainDataPath=[path]`: Path to your training data file(s), required when mode is NOT 'transform'.
+5. `-trainEvalDataPath=[path]`: Path to your data file(s) for training with evaluation. Optional.
+6. `-evalDataPath=[path]`: Path to your test(evaluation) data file(s), required when mode is NOT 'train'.
+7. `-modelPath=[path]`: Path to save model after training, or where to load model for transforming only. Required only when mode is 'transform'.
+8. `-overwrite=[true|false]`: Whether to overwrite the current model data under 'modelPath'. Default is false. You may need to set to true to avoid IOException when saving the model to a path already exists.
+9. `-hasHeader=[true|false]`: Indicate if your csv file has header.
 
+# Other Examples
```

Comment:
How about we change this heading.  Other examples makes me think there is a main example but this is it.  Maybe just Example Applications

Reply:
Well there's the "Getting Started Guides", which is what I consider the "main" example. But I can word it Example Applications.